sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -146,6 +146,12 @@ export class DataLakeFileClient extends DataLakePathClient {
     create(options?: FileCreateOptions): Promise<FileCreateResponse>;
     flush(position: number, options?: FileFlushOptions): Promise<PathUpdateResponse>;
     read(offset?: number, count?: number, options?: FileReadOptions): Promise<FileReadResponse>;
+    readToBuffer(buffer: Buffer, offset?: number, count?: number, options?: FileReadToBufferOptions): Promise<Buffer>;
+    readToBuffer(offset?: number, count?: number, options?: FileReadToBufferOptions): Promise<Buffer>;
+    readToFile(filePath: string, offset?: number, count?: number, options?: FileReadOptions): Promise<FileReadResponse>;
+    upload(data: Buffer | Blob | ArrayBuffer | ArrayBufferView, options?: FileParallelUploadOptions): Promise<PathUpdateResponse>;
+    uploadFile(filePath: string, options?: FileParallelUploadOptions): Promise<PathUpdateResponse>;
```

Comment:
The returned type is Promise<FileUploadResponse>?

Reply:
What's the problem?

Reply:
The tool is generating response as it's origin type. While its alias type looks better, anyway to generate with the type right in the code, e.g. here FileUploadResponse.

---
In reply to: [388189972](https://github.com/Azure/azure-sdk-for-js/pull/7659#discussion_r388189972) [](ancestors = 388189972)

Reply:
Make sense. But I don't know how.